frontend/src/component/changeRequest/ChangeRequest/Changes/Change/StrategyChange.tsx

```diff
@@ -25,7 +25,7 @@ export const ChangeItemWrapper = styled(Box)({
     alignItems: 'center',
 });
 
-const ChangeItemCreateEditWrapper = styled(Box)(({ theme }) => ({
+const ChangeItemCreateEditDeleteWrapper = styled(Box)(({ theme }) => ({
```

Comment:
Rename the component to make it clearer that it's being used also for deleted items now.